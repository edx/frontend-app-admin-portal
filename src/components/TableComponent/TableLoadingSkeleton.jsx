import React from 'react';
import Skeleton from 'react-loading-skeleton';

const TableLoadingSkeleton = () => (
  <>
    <div className="sr-only">Loading...</div>
    <Skeleton height={25} count={25} />
  </>
);

export default TableLoadingSkeleton;
