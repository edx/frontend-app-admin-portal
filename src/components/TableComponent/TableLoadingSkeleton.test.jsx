import React from 'react';
import renderer from 'react-test-renderer';
import TableLoadingSkeleton from './TableLoadingSkeleton';

describe('TableLoadingSkeleton', () => {
  it('renders a skeleton', () => {
    const tree = renderer
      .create((
        <TableLoadingSkeleton />
      ))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
