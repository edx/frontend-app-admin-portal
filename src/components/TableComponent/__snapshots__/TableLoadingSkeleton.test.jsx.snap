// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableLoadingSkeleton renders a skeleton 1`] = `
<div>
  <div
    className="sr-only"
  >
    Loading...
  </div>
  <span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 25,
        }
      }
    >
      ‌
    </span>
  </span>
</div>
`;
